.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-<!-- Thanks you for your contribution! -->
+<!-- Thank you for your contribution! -->
 
 <!-- Include if applicable: -->
 **Fixes**: _Tracking issues that this PR will close_
```

Comment:
Should we change this to just `Fixes #nnn <!-- tracking issues that this PR -->` so that we take advantage of Github's automatic linking?

Reply:
I'm fixing that